src/search/evaluator.cc

```diff
@@ -9,11 +9,24 @@
 using namespace std;
 
 
+Evaluator::Evaluator(bool use_for_reporting_minima,
```

Comment:
I'd add a newline after the opening bracket (here and in some other places below), but this is mainly a personal preference.